api/proto/teleport/legacy/client/proto/authservice.proto

```diff
@@ -2665,25 +2604,6 @@ service AuthService {
   // password token.
   rpc CreateResetPasswordToken(CreateResetPasswordTokenRequest) returns (types.UserTokenV3);
 
-  // CreateBot creates a new bot user.
-  //
-  // Deprecated: Use [teleport.machineid.v1.BotService] instead.
-  rpc CreateBot(CreateBotRequest) returns (CreateBotResponse) {
```

Comment:
FYI: I've been keeping deprecated/abandoned RPCs around, but replacing the impl with a custom error message (eg: https://github.com/gravitational/teleport/pull/37589/files#diff-0d992726e983ecd7ccd935e9769652763907ad1a88ae462050d2770e557d3747R2460).

Reply:
I don't think we do. The advantage of keeping the RPCs around is that, in the event of an old client, you get a readable message instead of a variation of "RPC whatever doesn't exist". The disadvantage is that there's a bit of "dead" code hanging around, accumulating over time.

For an external source, there's this Buf lint rule: https://buf.build/docs/breaking/rules#reserved_message_no_delete (that we don't use), although the explanation doesn't go a long way.